Entities/ACraft.h

```diff
@@ -573,6 +573,29 @@ enum
 	/// <param name="movableObjectToIgnore">A pointer to an MO which the Gibs and Attachables should not be colliding with.</param>
 	void GibThis(const Vector &impactImpulse = Vector(), MovableObject *movableObjectToIgnore = nullptr) override;
 
+	/// <summary>
+	/// Gets this ACraft's Hatch Open Sound. Ownership is not transferred!
+	/// </summary>
+	/// <returns>The SoundContainer for this ACraft's Hatch Open Sound.</returns>
+	SoundContainer* GetHatchOpenSound() const { return m_HatchOpenSound; }
+
+	/// <summary>
+	/// Sets this ACraft's Hatch Open Sound.
+	/// </summary>
+	/// <param name="newSound">The new SoundContainer for this ACraft's Hatch Open Sound</param>
```

Comment:
See comments in ACrab.h about `*` placement, method comment capitalization and `.`s at the end of method comments.